lib/ignore/ignore.go

```diff
@@ -387,6 +387,10 @@ func loadParseIncludeFile(filesystem fs.Filesystem, file string, cd ChangeDetect
 }
 
 func parseLine(line string) ([]Pattern, error) {
+	// We use native normalization internally, thus the patterns must match
+	// that to avoid false negative matches.
+	line = osutil.NativeFilename(line)
```

Comment:
Off the top of my head, does this convert `//comments` to `\\not comments` on Windows?

Reply:
Oh yes. I actually almost did the same mistake in the scanner, but then realized that was wrong. For some reason that realisation didn't propagate here, probably also need a fresher brain. Will fix tomorrow. And maybe have a look at all places where we do normalisation, if there's an opportunity for something like `osutil.NativeFilename`, just for unicode normalization only.

Reply:
I gave up on the osutil idea: We aren't really consistent in how we apply unicode normalization: In the same contexts we sometimes just leave paths alone, assuming they are already normalized, while in others we apply normalization again. That's a larger project to make that more consistent. I don't see anything that's wrong/buggy, just confusing (and potentially inefficient, though I doubt with any significant impact).